indexer/feature_utils.cpp

```diff
@@ -37,6 +37,9 @@ void GetMwmLangName(FeatureID const & id, StringUtf8Multilang const & src, strin
 void GetNames(FeatureID const & id, StringUtf8Multilang const & src, string & primary,
               string & secondary)
 {
+  if (src.IsEmpty())
```

Comment:
This breaks the old behavior - in any case primary and secondary must be cleared.